Addons/Entitas.Unity.VisualDebugging/Assets/Entitas.Unity.VisualDebugging/Entity/Editor/EntityDrawer.cs

```diff
@@ -262,9 +268,13 @@ struct ComponentInfo {
         }
 
         public static bool DidValueChange(object value, object newValue) {
+            var type = value.GetType();
+            var typeEquatable = getTypeEqualityComparer(type);
+
             return (value == null && newValue != null)
                 || (value != null && newValue == null)
-                || ((value != null && newValue != null && !newValue.Equals(value)));
+                || (value != null && newValue != null && typeEquatable != null && typeEquatable.Equals(value, newValue))
```

Comment:
Shouldn't that be
`typeEquatable != null && !typeEquatable.Equals(value, newValue))`